### PR TITLE
feat: emoji rendering

### LINKED ISSUE #304: [feat] add support to markdown emojis
**Feature**
Add support to github markdown emojis.

List of emojis: [https://gist.github.com/rxaviers/7360908](https://gist.github.com/rxaviers/7360908)

**Current implementation**
Currently emojis aren't rendered but instead shown as normal text.

image

**Expected result**

image

### PR DESCRIPTION
**Pull request by filippofinke**
This is just a proof of concept.

I think that we should use HTML5 entities but I am not sure how to go about it. Currently it's just replacing the code (for example `:lock:`) with the emoji assigned to id (🔒)

Wdyt @svedova?

It should resolve #304

image

### CODE REVIEW

File: src/utils/helpers/emojis.ts [hunk 1]
```diff
@@ -0,0 +1,364 @@
+export const renderEmojis = (text: string): string => {
+  if (typeof text !== "string") return text;
+
+  for (let emoji of emojis) {
+    text = text.replace(emoji.code, emoji.emoji);
```
[the remark this answers is not in the document]
Comment (filippofinke) [reply src/utils/helpers/emojis.ts:5]: Yes, I agree with you, it's just a proof of concept.

I haven't found a library that does this.

[the remark this answers is not in the document]
Comment (svedova) [reply src/utils/helpers/emojis.ts:5]: How about this one? npmjs.com/package/node-emoji

[the remark this answers is not in the document]
Comment (filippofinke) [reply src/utils/helpers/emojis.ts]: Yes, I think that node-emoji is the one to go with.

**Review by svedova — APPROVED** [review]
Huge thanks @filippofinke for working on this 🎉

**Pull request merged**